src/NuGetPackageConfigConverter/Services/PackageManagerConverter.cs

```diff
@@ -272,4 +392,53 @@ private static ProjectItem GetProjectItem(ProjectItems items, string name)
             return null;
         }
     }
+
+
+
+    static class ProjectExtensions
```

Comment:
can you move these types to their own files?

Reply:
yep